eng/pipeline/variables/common.yml

```diff
@@ -26,11 +24,12 @@ variables:
     # In the nightly branch, publish to ACR only. The prefix doesn't need to match MAR expectations.
     value: nightly/
 
+# Don't publish readme: Microsoft Go images aren't on Docker Hub.
 - name: publishReadme
   value: false
-# https://github.com/microsoft/go/issues/157 tracks enabling publishImageInfo.
+# Publish image info to https://github.com/dotnet/versions/tree/main/build-info
```

Comment:
Is this location a hang over from inheriting their infra?

Reply:
Yeah, it's the repo where .NET Docker posts their build-info as well. There are no roadblocks preventing us from creating our own repository, another bot account, etc., but it's more practical to share.